content/components/recipes/details/ExperimenterDetails.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+
+import CollapsibleSection from "devtools/components/recipes/details/CollapsibleSection";
+import { useExperimenterDetailsData } from "devtools/contexts/experimenterDetails";
+
+const ExperimenterDetails: React.FunctionComponent<{}> = () => {
+  const data = useExperimenterDetailsData();
+  const {
+    publicDescription,
+    proposedStartDate,
+    proposedEndDate,
+    proposedDuration,
+    startDate,
+    endDate,
+    variants,
+  } = data;
+
+  return (
+    <CollapsibleSection
+      headerButtons={<></>}
+      title={
+        <>
+          <h6>Experimenter Details</h6>
+        </>
+      }
+    >
+      <div className="py-1 pl-4">
+        <div className="mt-4">
+          <strong>Public description</strong>
+          <p>{publicDescription}</p>
+        </div>
+        <div className="mt-4" data-testid="details-proposed-schedule">
+          <strong>Proposed schedule</strong>
+          <p>
+            {proposedStartDate?.toDateString()} →{" "}
+            {proposedEndDate?.toDateString()} ({proposedDuration} days)
+          </p>
+        </div>
+        <div className="mt-4">
+          <strong>Actual Schedule</strong>
+          <p>
+            {startDate ? startDate.toDateString() : "N/A"}
+            {endDate ? " → " + endDate.toDateString() : ""}
+          </p>
+        </div>
+        <div className="mt-4">
+          <strong>Branches</strong>
+          {variants.length ? (
```

Comment:
I don't think it is useful to list the variants here, as it spreads associated data across multiple places. The variant names are more useful when combined with the rest of the data associated with each. We have that info already in the "Branches" section of the relevant action arguments (Experimenter's variants are Normandy's branches). I think that instead we should try and associate these variant names with the branches where they already are.

Reply:
Can we spin off an issue to follow-up on that? 

It looks like I will have to go through every kind of action and merge details there. I think this PR is big enough and would feel better to first merge this and go on with smaller stuff...

Reply:
Yeah, lets spin that off. I'm looking forward to getting this merged too.